lib/matobjplist.gi

```diff
@@ -1348,10 +1348,6 @@ InstallMethod( AddMatrixRowsRight, "for a mutable IsRowListMatrix, one row numbe
 InstallMethod( SwapMatrixRows, "for a mutable IsRowListMatrix, one row number, second row number",
   [ IsRowListMatrix and IsMutable, IsInt, IsInt ],
   function( mat, row1, row2 )
-    local temp;
-  
-    temp := mat[row1];
-    mat[row1] := mat[row2];
-    mat[row2] := temp;
+    mat{[row1,row2]} := mat{[row2,row1]};
 
```

Comment:
This gives us an error:

Error, List Assignments: <rhss> must be a dense list (not a positional object) in
  mat{[ row1, row2 ]} := mat{[ row2, row1 ]}; at .../GAP/gap/lib/matobjplist.gi:1351 called from 
<function "SwapMatrixRows for a mutable IsRowListMatrix, one row number, second row number">( <arguments> )
 called from read-eval loop at *stdin*:6
type 'quit;' to quit to outer loop

Reply:
This problem shows a bug in GAP, see issue #4533.